org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -422,6 +430,29 @@ SsoViewerPlus:
       - arn:aws:iam::aws:policy/AWSLambda_ReadOnlyAccess
       - arn:aws:iam::aws:policy/AWSBillingReadOnlyAccess
 
+# Role for a user that manages an application deployed to AWS
+SsoApplicationManager:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/SSO/aws-sso.njk
+  StackName: !Sub '${resourcePrefix}-${appName}-application-manager'
+  StackDescription: 'Read-only role plus access to secrets manager'
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: '*'
+  Parameters:
```

Comment:
`principalId` is a required parameter which means you'll need to create an AWS SSO group for this as well.  the group can be empty.

Reply:
OK to just use `adminGroup`? This avoids creating an empty placeholder.

Reply:
good idea.